packages/frontend/apps/electron/scripts/common.ts

```diff
@@ -34,6 +34,14 @@ export const config = (): BuildOptions => {
     define['process.env.GITHUB_SHA'] = `"${process.env.GITHUB_SHA}"`;
```

Comment:
<!--__GRAPHITE_HTML_TAG_START__--><i class='graphite__hidden'>[Re: line +23]<br/><br/></i>

<!--__GRAPHITE_HTML_TAG_END__-->damn, miss<!--__GRAPHITE_HTML_TAG_START__--><span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/toeverything/AFFiNE/8286?utm_source=unchanged-line-comment">Graphite</a>.</span>

<!--__GRAPHITE_HTML_TAG_END__-->